__tests__/ExpensiMark-HTML-test.js

```diff
@@ -23,7 +23,7 @@ test('Test multi-line bold markdown replacement', () => {
 
 // Sections starting with > are successfully wrapped with <blockquote></blockquote>
 test('Test quote markdown replacement', () => {
-    const quoteTestStartString = '&gt;This is a *quote* that started on a new line.\nHere is a &gt;quote that did not\n```\nhere is a codefenced quote\n>it should not be quoted\n```';
```

Comment:
Why are we updating this test? I'm guessing that this might break some Web-E stuff since Web-E sends stuff with HTML entities.

Reply:
I need to change it because we change from `safeEscape` to `escape`, so the `&gt;` will become `&amp;gt;` and fails the test. I also actually afraid it would break stuff on other App that use this too, maybe we should wait @yuwenmemon testing on Web repo.